sdk-base/src/main/java/com/mapbox/maps/threading/AnimationThreadController.kt

```diff
@@ -0,0 +1,112 @@
+package com.mapbox.maps.threading
+
+import android.os.Handler
+import android.os.HandlerThread
+import android.os.Looper
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.logW
+
+/**
+ * Experimental controller singleton allowing to use background thread for all Mapbox animations.
```

Comment:
@kiryldz Will you add a row to the `README.md`

https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto#compatibility-with-maps-sdk-v10

| `AnimationThreadController` | :white_check_mark:  | v10.9+ |

or 

| Xiaomi animation support | :white_check_mark:  | v10.9+ |

Reply:
done

Reply:
Nit pick, but the ordering is not right.

The list of compatibility version are in descending order. This row was added to the top so the list is no longer sorted

Reply:
feel free to arrange this as needed in tailwork PRs